packages/modules/common/store/_chargepoint.py

```diff
@@ -31,6 +31,7 @@ def set(self, state: ChargepointState) -> None:
         pub_to_broker("openWB/set/chargepoint/" + str(self.num) + "/get/imported", state.imported, 2)
         pub_to_broker("openWB/set/chargepoint/" + str(self.num) + "/get/exported", state.exported, 2)
         pub_to_broker("openWB/set/chargepoint/" + str(self.num) + "/get/power", state.power, 2)
+        pub_to_broker("openWB/set/chargepoint/" + str(self.num) + "/get/powers", state.powers, 2)
```

Comment:
Fehlt hier nicht noch die Frequenz?
```suggestion
        pub_to_broker("openWB/set/chargepoint/" + str(self.num) + "/get/powers", state.powers, 2)
        pub_to_broker("openWB/set/chargepoint/" + str(self.num) + "/get/frequency", state.frequency, 2)
```